docs/how-to-docs/cert-manager.md

```diff
@@ -141,57 +176,36 @@ $ kubectl -n project get certificate
+Please note that as part of the implementation of cert-manager v0.13.1, a `GlobalNetworkPolicy` object managing ingress traffic for `http01` challenges has been deployed.
 
-- creating a order resource within the namespace.
-- a controller pick up the order and creates a challenge resource from it, creating an ephermal pod, service and ingress in your namespace and routing the `/.well-known/acme-challenge/`.
-- the controller checks the path/s has been provision via the ingress beforehand and validates the order, the certificate controller is then free to inform letsencrypt to call us back.
-- it continues to probe the request and once validated, pulls down the certificate.
+This means that you no longer need to have a `NetworkPolicy` in your namespaces allowing ingress traffic from port 8089 to the ephemeral pods that cert-manager creates to handle the `http01` challenge.
 
-**IMPORTANT** All this requires the user's to add a network policy to permit the callback, as ACP by default denies all traffic.
+### **As a developer I want to retrieve a certificate for a service behind the vpn, or simply wish to use the DNS validation**
 
-```YAML
-# This default policy permits access for LetsEncrypt to resolve the http01 challenges from the ACME pods
-kind: NetworkPolicy
-apiVersion: networking.k8s.io/v1
-metadata:
-  name: permit-certmanager-acme
-spec:
-  policyTypes:
-  - Ingress
-  podSelector:
-    matchExpressions:
-    - {key: certmanager.k8s.io/acme-http-domain, operator: Exists}
-    - {key: certmanager.k8s.io/acme-http-token, operator: Exists}
-  ingress:
-  - from:
-    - namespaceSelector:
-        matchLabels:
-          name: ingress-external
-    - podSelector:
-        matchLabels:
-          name: ingress
-    ports:
-    - protocol: TCP
-      port: 8089
-```
+When a site is internal / behind the vpn, in order to handle the challenge you need to switch to using a DNS challenge.
 
-#### **As a developer I want to retrieve a certificate for a service behind the vpn, or simple wish to use the DNS validation**
+This is done by adding the following to your ingress resource:
 
-As stated above in order to retrieve a certificate for sites which are not externally facing we need to switch to a DNS challenge. Please ensure you have contacted the ACP team before attempting this as the correct permission need to exist to permit the cert-manager to add records to the domain.
+- annotation `cert-manager.io/enabled: "true"`
+- label `cert-manager.io/solver: route53`
+
+**Very Important**: in order to successfully switch to a DNS challenge, please ensure you have contacted the ACP team before attempting this for the first time on your sub-domain as the correct permissions need to exist to permit cert-manager to add records to the domain.
```

Comment:
I believe policy-admission performs a check on this and rejects ingress resources referencing a domain which is not managed by Route53 in the relevant account. This should be checked again following the migration to jetstack's cert manaager